File: CursoLogica/Linguagens/Python/Atividades/menor_de_tres.py
valor1 = int(input("Digite o primeiro valor: "))
valor2 = int(input("Digite o segundo valor: "))
valor3 = int(input("Digite o terceiro valor: "))

if valor1 < valor2 and valor1 < valor3:
    print(f"MENOR = {valor1}")
elif valor2 < valor3:
    print(f"MENOR = {valor2}")
else:
    print(f"MENOR = {valor3}")

File: CursoLogica/Linguagens/C/Atividades/menor_de_tres.c
#include <stdio.h>

int main() {

    int x, y, z, menor;

    printf("Digite o primeiro valor: ");
    scanf("%d", &x);

    printf("Digite o segundo valor: ");
    scanf("%d", &y);

    printf("Digite o terceiro valor: ");
    scanf("%d", &z);

    if (x < y && x < z) {
        menor = x;
    }
    else if (y < z) {
        menor = y;
    }
    else {
        menor = z;
    }

    printf("MENOR = %d", menor);

    return 0;
}
